contrib/docker/Dockerfile.fpm-alpine

```diff
@@ -0,0 +1,94 @@
+FROM php:8.1-fpm-alpine AS app_php
+
+ARG UID=1000
+ARG GID=1000
+
+RUN set -eux; \
+	addgroup -g $GID pixelfed; \
+    adduser -u $UID -D -G pixelfed pixelfed; \
+    mkdir -p /home/web; chown $UID:$GID /home/pixelfed; \
+    mkdir -p /srv/app; chown $UID:$GID /srv/app; \
+    mkdir -p /var/run/php; chown $UID:$GID /var/run/php
+
+# Add php extensions installer: https://github.com/mlocati/docker-php-extension-installer
+COPY --from=mlocati/php-extension-installer --link /usr/bin/install-php-extensions /usr/local/bin/
+
+# Add composer
+COPY --from=composer/composer:2-bin --link /composer /usr/bin/composer
+
+# Add healthcheck
+ADD contrib/docker/php-fpm-healthcheck /usr/local/bin/php-fpm-healthcheck
+RUN chmod a+rx /usr/local/bin/php-fpm-healthcheck
+
+# Add entrypoint
+COPY --link contrib/docker/fpm-alpine-entrypoint.sh /usr/local/bin/pixelfed-entrypoint
+RUN chmod +x /usr/local/bin/pixelfed-entrypoint
+
+# Locales
+WORKDIR /tmp
+ENV MUSL_LOCALE_DEPS cmake make musl-dev gcc gettext-dev libintl
+ENV MUSL_LOCPATH /usr/share/i18n/locales/musl
+ADD https://gitlab.com/rilian-la-te/musl-locales/-/archive/master/musl-locales-master.zip /tmp/musl-locales-master.zip
```

Comment:
Some cases you download, others you vendorp I would just change everything to be a download imo.